torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -770,7 +770,6 @@ def test_register_rpc_backend_and_set_and_start_rpc_backend(
             rpc_backend_options=self.rpc_backend_options,
         )
 
-    @dist_init(setup_rpc=False)
```

Comment:
Was this intentional?

Reply:
this is not intentional, my bad. Thanks for spotting this.